### PR TITLE
Linda reviews

### PR DESCRIPTION
**Pull request by liber09**

### CODE REVIEW

**Review by Rasmus3liasson — APPROVED** [review]
Ser bra ut där recensionerna visas enligt beskrivning i uppgiften. Samt att testen genomförs med godkänt resultat👍🏻

**Review by a-jaxell — APPROVED** [review]
Ser bra ut!

Slimmat och koncentrerat, bra jobbat!

**Review by KRQQN — APPROVED** [review]
Snyggt och välstrukturerat!
ser att du hittat en miss i min server.js med :D 
bra jobbat!

File: server/loadReviews.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    const res = await fetch(reviewbaseUrl + "?filters[movie]=" + movieID+"&pagination[pageSize]=5&pagination[page]="+page);
+    //Convert to JSON
+    
+    //Get JSON data and metadata for pagination and return info
+    const data = await res.json();
+    const pages = data.meta;
+      return {
+        reviews: data,
+        pages: pages
+      };
```
Comment (richardolsson) [inline server/loadReviews.js:17]: Here you're getting content from the `data` object, and returning it alongside the original `data` object (called `reviews`). This means that the returned object contains the same data twice, once as `reviews.meta` and once as `pages`.

File: test/loadMovies.test.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import request from "supertest";
 import app from "../server/server.js";
+import { loadReviews } from "../server/loadReviews.js";
```
Comment (richardolsson) [inline test/loadMovies.test.js:3]: This is being imported here, but not used anywhere in the file.

File: server/server.js [hunk 3]
```diff
@@ -39,6 +41,24 @@ app.get("/movies/:id", async (req, res) => {
...
+});
+
+//Get reviews, takes movieId and pageNumber as parameters.
+//calls server function loadReviewsForPageX.
+//Then sends response back to frontend
+app.get("/reviews/:id/", async (req, res) => {
+  const page = req.query.page;
+  const reviews = await loadReviews(req.params.id,page);
+  res.send(reviews)
+});
```
Comment (richardolsson) [inline server/server.js:60]: I think this is the exact same code repeated twice, i.e. both on lines 44-51 and 53-60.

**Review by richardolsson — COMMENTED** [review]
Great work! I've left som feedback below. Only very minor things, and not anything that needs to be changed for this assignment. Besides those isolated things, there are also a fair amount of formatting inconsistencies. I recommend everyone to use automatic formatting, to avoid those!

**Pull request merged**